baselines/run.py

```diff
@@ -10,14 +10,18 @@
 from baselines.common.vec_env import VecFrameStack, VecNormalize, VecEnv
 from baselines.common.vec_env.vec_video_recorder import VecVideoRecorder
 from baselines.common.cmd_util import common_arg_parser, parse_unknown_args, make_vec_env, make_env
-from baselines.common.tf_util import get_session
 from baselines import logger
 from importlib import import_module
 
 try:
     from mpi4py import MPI
 except ImportError:
     MPI = None
+# Fix for loading Roboschool environments
+try:
+    from OpenGL import GLU
```

Comment:
why is this necessary? If anything, let's rather move it to roboschool package

Reply:
You're right...should just be a fix on roboschool side.